### DISCUSSION TITLE
alguém saberia me dizer o porque o python não esta executando este comando?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#123522] [New to GitHub]
ele executa mas não clica na imagem
alguém sabe o porque isto esta acontecendo?
![Captura de tela 2024-05-12 135036](https://github.com/community/community/assets/169319645/b8ea166e-dda5-42ca-88c1-d4d41491de09)

**Comment by davevad93** [comment]
Hi @cavaloXdeXpinga ,

Try with this code and see if it works:

```python
import pyautogui
import time

procurar = "sim"

while procurar == "sim":
    try:
        img = pyautogui.locateCenterOnScreen("Capturar1.png", confidence=0.9)
        if img is not None:
            pyautogui.click(img)
            time.sleep(1)
        else:
            print("não encontrado")
            time.sleep(1)
    except Exception as e:
        print("Error: ", e)
        time.sleep(1)
```

**Comment by grandmas-favorite** [comment]
Hi @cavaloXdeXpinga, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
